content/en/core/releases/4.10.0.md

```diff
@@ -0,0 +1,118 @@
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+This release includes two improvements to [Aggregate Targets](https://docs.communityhealthtoolkit.org/apps/features/supervision/#chw-aggregate-targets). 
+
+- Users can now view data for the previous reporting period
+- Users [assigned multiple places](https://docs.communityhealthtoolkit.org/core/releases/4.9.0/#uiux-changes) can now see Aggregate Targets
+
+
+## Highlights
+
+### Aggregate Targets can now be viewed for the previous month
+The CHT now supports viewing Aggregate Targets for the previous month. All users who have access to Aggregate Targets will see a Filter option that allows them to choose between the current month and previous month. This affects both “monthly” and “all time” [widgets](https://docs.communityhealthtoolkit.org/apps/features/targets/#types-of-widgets). Viewing the previous month’s data for an “all time” widget (which is cumulative) will show the data as it were at the end of the previous month. When “Previous Month” is selected, a visual cue (the month name) will be displayed both in the list view (as a breadcrumb) and the detail view (as the column header).
+
+##### Details
+- “Current month” will always be shown by default
+- The “Reporting Period” label in the filter sidebar uses the  `analytics.target.aggregates.reporting_period` translation key
+- This filter is shown for all users who have access to aggregate targets, you don’t need to do anything to enable it (and there is no way to disable it)
+
+{{< figure src="../images/4.10.0-aggregate-targets.png">}}
+
+### Aggregate Targets for users assigned to multiple places
+The CHT also now supports Aggregate Targets for users that are assigned to multiple places. If a user has multiple places (added in 4.9.0), they will see an additional filter option that allows them to choose which place they want to view. There is also a visual cue on each Target to clearly indicate which place is being viewed.
+
+_NOTE: Users who are assigned only one place will not see the additional place filter option, but they will continue to see the new Reporting Period filter option._
+
+##### Details
+- The list of places will be shown alphabetically
+- The first place in the list will be selected by default
+- Only one place can be selected at a time (it’s not a multi-select)
+- The label / heading for the list of places in the filter sidebar is the [name_key](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/hierarchy/#app_settingsjson-contact_types) of the place `contact type` as set in [app_settings](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/)
+- This filter is shown for all users who have access to aggregate targets and are configured with multiple places, you don’t need to do anything to enable it (and there is no way to disable it)
+
+{{< figure src="../images/4.10.0-aggregate-targets-2.png">}}
+
```

Comment:
```suggestion
### Add xpath functions for Luhn identifiers

[Luhn numbers](https://en.wikipedia.org/wiki/Luhn_algorithm) are used in a range of identification schemes and include a check digit to allow validation so they can be checked for common mistakes like typos. Including this in xpath means this can be checked while the form is being filled out to detect mistakes before the form is completed. More information is available in the [xpath documentation](https://docs.communityhealthtoolkit.org/apps/reference/forms/app/#chtvalidate-luhn).

[#9227](https://github.com/medic/cht-core/issues/9227): Include xpath function for validation of Luhn identifiers

```